play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/Aliases.scala

```diff
@@ -30,8 +30,10 @@ trait Aliases {
   val CheckProgress = viewmodels.accountmenu.CheckProgress
 
   val Cy = viewmodels.language.Cy
+  val CyServiceNavigation = viewmodels.language.CyServiceNavigation
 
   val En = viewmodels.language.En
+  val EnServiceNavigation = viewmodels.language.EnServiceNavigation
```

Comment:
Two additional models for language created to avoid changing what we have currently. The difference is in name. `ENG` instead of `English` and `CYM` instead of `Cymraeg`.

Probably as future implementation we can combine those into one model with 3 fields